docs/guides/cache/index.md

```diff
@@ -0,0 +1,97 @@
+
+- `s3` AWS cache, that uploads the build cache to an [AWS S3 bucket](https://aws.amazon.com/s3/)
+  (unreleased).
+
+  ([guide](./s3.md))
+
+- `azblob` Azure cache, that uploads the build cache to [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/)
+  (unreleased).
+
+  ([guide](./azblob.md))
+
+To use any of the above backends, you first need to specify it on build with
+the [`--cache-to`](../../reference/buildx_build.md#cache-to) option to export
+the cache to your storage backend of choice, then use the [`--cache-from`](../../reference/buildx_build.md#cache-from)
+option to import the cache from the storage backend into the current build.
+Unlike the local BuildKit cache (which is always enabled), **all** of the cache
+storage backends have to be explicitly exported to and then explicitly imported
+from. Note that all cache exporters except for the `inline` cache, require
+[selecting an alternative Buildx driver](../drivers/index.md).
+
+For example, to perform a cache import and export using the [`registry` cache](./registry.md):
+
+```console
+$ docker buildx build --push -t <user>/<image> \
+  --cache-to type=registry,ref=<user>/<cache-image> \
+  --cache-from type=registry,ref=<user>/<cache-image> .
+```
+
+> *Warning*
+>
+> As a general rule, each cache writes to some location - no location can be
+> written to twice, without overwriting the previously cached data. If you want
+> to maintain multiple separately scoped caches (e.g. a cache per git branch),
+> then ensure that you specify different locations in your cache exporters.
+
+While [currently](https://github.com/moby/buildkit/pull/3024) only a single
+cache exporter is supported, you can import from as many remote caches as you
+like. For example, a common pattern is to use the cache of both the current
+branch as well as the main branch (again using the [`registry` cache](./registry.md)):
+
+```console
+$ docker buildx build --push -t <user>/<image> \
+  --cache-to type=registry,ref=<user>/<cache-image>:<branch> \
+  --cache-from type=registry,ref=<user>/<cache-image>:<branch> \
+  --cache-from type=registry,ref=<user>/<cache-image>:master .
+```
+
+BuildKit supports multiple cache backends, however, they do support different
+sets of features.
+
+| Feature                             | `inline` | `registry` | `local` |    `gha`     |    `s3`    |  `azblob`  |
+|:------------------------------------|:--------:|:----------:|:-------:|:------------:|:----------:|:----------:|
+| **Status**                          |  stable  |   stable   | stable  | experimental | unreleased | unreleased |
+| **Compatible with `docker` driver** |    ✅     |     ❌      |    ❌    |      ❌       |     ❌      |     ❌      |
+| **Compatible with all exporters**   |    ❌     |     ✅      |    ✅    |      ✅       |     ✅      |     ✅      |
+| **External authentication**         |    ❌     |     ❌      |    ❌    |      ✅       |     ✅      |     ✅      |
```

Comment:
`External authentication` looks confusing to me. `inline` and `registry` auth is not required when fetching cache but exporting is.

Maybe we should remove this table and stipulate support and auth mode on each cache exporter page. WDYT?

Reply:
Agreed, not sure the table works very well, we've also got the support information above in the list, can duplicate it on the pages.